riscv-page-tables/src/page_table.rs

```diff
@@ -548,16 +580,23 @@ impl<T: PagingMode> PageTableInner<T> {
 
     /// Locks the invalid leaf PTE mapping `vaddr`, filling in any missing intermediate page tables
     /// using `get_pte_page`.
-    fn lock_4k_leaf_for_mapping(
+    /// Locks the invalid leaf PTE mapping `vaddr`, filling in any missing intermediate page tables
+    /// using `get_pte_page`.
+    fn lock_general_leaf_for_mapping(
```

Comment:
nit: drop the 'general', it's cleaner.

`lock_leaf_for_mapping`

Reply:
fixed